__tests__/SelfMailersApi.spec.ts

```diff
@@ -65,49 +65,85 @@ describe("smApi", () => {
     let previousUrl = "";
     beforeAll(async () => {
       const smApi = new SelfMailersApi(CONFIG_FOR_INTEGRATION);
-      // ensure there are at least 3 cards present, to test pagination
-      const sfm1 = new SelfMailerEditable({
-        to: ADDRESSES_EDITABLE[1],
-        from: ADDRESSES_EDITABLE[2],
-        inside: FILE_LOCATION_6X18,
-        outside: FILE_LOCATION_6X18,
-        use_type: "operational",
-      });
-      const sfm2 = new SelfMailerEditable({
-        to: ADDRESSES_EDITABLE[3],
-        from: ADDRESSES_EDITABLE[6],
-        inside: FILE_LOCATION_6X18,
-        outside: FILE_LOCATION_6X18,
-        use_type: "operational",
-      });
-      const sfm3 = new SelfMailerEditable({
-        to: ADDRESSES_EDITABLE[4],
-        from: ADDRESSES_EDITABLE[5],
-        inside: FILE_LOCATION_6X18,
-        outside: FILE_LOCATION_6X18,
-        use_type: "marketing",
-      });
-      await smApi.create(sfm1);
-      await smApi.create(sfm2);
-      await smApi.create(sfm3);
-
-      const response = await smApi.list();
-      if (response && response.next_url) {
+
+      // Create enough self-mailers to ensure pagination works
+      // We'll create more than the default page size to guarantee pagination
+      const selfMailersToCreate = [
+        new SelfMailerEditable({
+          to: ADDRESSES_EDITABLE[1],
+          from: ADDRESSES_EDITABLE[2],
+          inside: FILE_LOCATION_6X18,
+          outside: FILE_LOCATION_6X18,
+          use_type: "operational",
+        }),
+        new SelfMailerEditable({
+          to: ADDRESSES_EDITABLE[3],
+          from: ADDRESSES_EDITABLE[6],
+          inside: FILE_LOCATION_6X18,
+          outside: FILE_LOCATION_6X18,
+          use_type: "operational",
+        }),
+        new SelfMailerEditable({
+          to: ADDRESSES_EDITABLE[4],
+          from: ADDRESSES_EDITABLE[5],
+          inside: FILE_LOCATION_6X18,
+          outside: FILE_LOCATION_6X18,
+          use_type: "marketing",
+        }),
+        new SelfMailerEditable({
+          to: ADDRESSES_EDITABLE[0],
+          from: ADDRESSES_EDITABLE[1],
+          inside: FILE_LOCATION_6X18,
+          outside: FILE_LOCATION_6X18,
+          use_type: "operational",
+        }),
+        new SelfMailerEditable({
+          to: ADDRESSES_EDITABLE[2],
+          from: ADDRESSES_EDITABLE[3],
+          inside: FILE_LOCATION_6X18,
+          outside: FILE_LOCATION_6X18,
+          use_type: "marketing",
+        }),
+        new SelfMailerEditable({
+          to: ADDRESSES_EDITABLE[4],
+          from: ADDRESSES_EDITABLE[6],
+          inside: FILE_LOCATION_6X18,
+          outside: FILE_LOCATION_6X18,
+          use_type: "operational",
+        }),
+      ];
```

Comment:
Similar to above; use parameterized fixtures for the test setup and create self mailers in parallel.